### DISCUSSION TITLE
Interceptor Timeout

### DISCUSSION THREAD

**Discussion by joshua-blickensdoerfer** [tektoncd/triggers#1451] [Q&A] [answered]
Hello,

i have written a custom Cluster interceptor. In the eventlistener log i can see that there is a timeout after a few seconds

"logger":"eventlistener","caller":"sink/sink.go:381","msg":"Post \"http://custom-svc.tekton-framework.svc:80\": net/http: timeout awaiting response headers"

Is there any way to increase the timeout duration for custom interceptors? I've tried increasing the value of "'-el-readtimeout" and "-el-httpclient-readtimeout" this did not seem to have any effect.

Kind regards
Joshua

**Comment by dibyom** [comment]
I turned this into a feature request for us to track: https://github.com/tektoncd/triggers/issues/1452

**Comment by jmcshane** [comment, marked as answer]
Until this is implemented on the eventlistener, the argument you need to increase on the eventlistener is `--httpclient-responseheadertimeout`
